residentevil2remake/__init__.py

```
@@ -338,63 +336,28 @@ def create_items(self):
                     self.multiworld.early_items[self.player][item_name] = item_qty
    
 
-        # check the "Oops! All Rockets" option. From the option description:
-        #     Enabling this swaps all weapons, weapon ammo, and subweapons to Rocket Launchers. 
+        # check the "Oops! All ____" option. From the option description:
```

The consolidating parts in this file from here on, I do agree with. These options were copy-pasta doing the same things, so consolidating this is good.

---

Yeah bare minimum maybe making this its own function would be worth doing. Maybe it can yell at a player for ticking multiple oops options?

---

Yeah, it should probably throw an error during gen if they have more than one Oops enabled at a time. Good call :smile: (Applies to both comments, haha)

Oh, we also currently just return out when processing weapon rando and any Oops options are enabled, so could probably throw the same error there, too. Error can either be added in this PR or after, whatever works